### ISSUE TITLE
Go To Definition/Peek Definition Got Error

### ISSUE THREAD

**Issue by 4406arthur** [pgourlain/vscode_erlang#95]
Hi, I got error msg as follow, even on same module file func

```
[Error - 10:35:17] Request textDocument/hover failed.
  Message: Handler error
  Code: -32001 
[Error - 10:35:18] Request textDocument/definition failed.
  Message: Handler error
  Code: -32001
```

**Comment by pgourlain** [comment]
Hi,

Could you activate traces ?
to enable trace : "erlang.verbose": true in your usersettings
Then traces are displayed in 'Output' Pane, then choose 'Erlang Language Server' view

**Comment by 4406arthur** [comment]
something like
```

**Comment by wojteksurowka** [comment]
@4406arthur Should be fixed by PR #96, please retest when 0.4.5 is available.

**Comment by wojteksurowka** [comment]
Fixed in the current version, can be closed.
